mandatory/4-tax.js

```diff
@@ -5,7 +5,10 @@
   Sales tax is 20% of the price of the product.
 */
 
-function calculateSalesTax() {}
+function calculateSalesTax(amount) {
+  result = (20/100*amount) + amount;
```

Comment:
This code will give you ReferenceError: result is not defined.
Remember to use let or const key words when you declare a variable. 
Like: `const result = (20/100*amount) + amount;`